src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final int INT_BOUND = 100;
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
```

Comment:
Not fixed, external services we rely on are commonly private final (further such practice will benefit from dependency injection pattern)
```suggestion
    private static final int INT_BOUND = 100;
    private final Random random = new Random();  
    private final ColorSupplier colorSupplier = new ColorSupplier();
```